pootle/static/js/shared/models/project.js

```diff
@@ -34,6 +34,10 @@ export const Project = Backbone.Model.extend({
     return l(['', 'projects', this.get('code'), ''].join('/'));
   },
 
+  getAbsoluteUrl() {
```

Comment:
I see this is amended by https://github.com/translate/pootle/pull/5314/commits/1dae3d3a4379d3b3cb9767aadc81390d5eb3cc8d Please squash these commits.